verification/curator-service/ui/src/components/LinelistCaseRow.test.tsx

```diff
@@ -15,11 +15,21 @@ it('renders provided case', () => {
           case={
             {
               _id: id,
-              outcome: outcome,
-              eventSequence: {
-                confirmed: date
-              }
-            }}
+              events: [
+                {
+                  name: 'confirmed',
+                  date: {
+                    range: {
+                      start: date,
+                    },
+                  },
+                }
+              ],
+              importedCase: {
```

Comment:
huh, ideally we shouldn't be filling anything in imported cases from this UI, this is only for stuff that comes out of the existing CSV. Not sure why outcome was dropped, we should check with @khmoran .

Reply:
Turns out outcome was in the schema (https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/data-serving/data-service/src/model/case.ts) it just wasn't in the sample data. Added to the sample data and reverted to using the string outcome.

Reply:
outcome is in the mongoose schema but not in the mongodb schema, @khmoran can you please tell us which one is correct?

https://github.com/open-covid-data/healthmap-gdo-temp/blob/master/data-serving/data-service/schemas/cases.schema.json

Reply:
oops sorry y'all -- the mongodb schema + example case are up-to-date

was working on a PR for the dataserver to make it backward compatible with the fields the UI was using, but I should have done that atomically. from now on will make sure I don't break you!

it's on my list to separate the storage format from the dataserver API format

Reply:
Spoke with Kelly and decided to keep using the importedCase. The schema is going to change a bit and we don't want to keep updating the UI to match the changing schema. So for now we'll rely on the imported case.